packages/graphql-language-service-server/package.json

```diff
@@ -50,7 +53,6 @@
     "graphql-language-service": "^5.3.0",
     "lru-cache": "^10.2.0",
     "mkdirp": "^1.0.4",
-    "node-abort-controller": "^3.0.1",
```

Comment:
i think this is not needed according to [docs](https://github.com/southpolesteve/node-abort-controller#node-abort-controller) it's polyfill for node <14.70